apps/opencs/model/world/refidadapterimp.cpp

```diff
@@ -1373,10 +1373,8 @@ QVariant CSMWorld::CreatureAttackRefIdAdapter::getNestedData (const RefIdColumn
 
     if (subColIndex == 0)
         return subRowIndex + 1;
-    else if (subColIndex < 3) // 1 or 2
+    else // 1 or 2
```

Comment:
I would prefer to keep the condition. If there every arises the need to add more columns the removal of the condition because a possible source of a bug.